pkg/operator/ceph/csi/spec.go

```diff
@@ -78,6 +78,12 @@ const (
 	DefaultAttacherImage    = "quay.io/k8scsi/csi-attacher:v1.2.0"
 	DefaultSnapshotterImage = "quay.io/k8scsi/csi-snapshotter:v1.2.0"
 
+	// toleration and node affinity
+	tolerationEnv    = "CSI_TOLERATION"
+	tolerationKeyEnv = "CSI_TOLERATION_KEY"
+	tolerationsEnv   = "CSI_TOLERATIONS"
```

Comment:
Why do we have both `CSI_TOLERATION` and `CSI_TOLERATIONS`? We could just go with `CSI_TOLERATIONS`, right?

Reply:
I referred agent toleration for this one, can we remove `CSI_TOLERATION` then the user has to provide `operator` `effect` and `Key` i can make the changes.

let me check if I can make few defaults if some values are not provided